src/AltinnCore/Designer/Controllers/DeployController.cs

```diff
@@ -315,28 +315,28 @@ private async Task<bool> RegisterApplicationInStorage(string applicationOwnerId,
             }
         }
 
-        private ApplicationMetadata GetApplicationMetadata(string applicationId, string versionId)
+        private Application GetApplicationMetadata(string appId, string versionId)
```

Comment:
Should the working be cleaned up in this method? 
GetApplicationMetadata -> GetApplication 
defaultAppForm -> DefaultAppElement (?) Not quite sure about this.. but my understanding is that we are moving away from form

Reply:
Have renamed methods and variables